src/Surfnet/ServiceProviderDashboard/Application/ViewObject/EntityConnection.php

```diff
@@ -47,13 +50,22 @@ public function listConnected(): array
         return $list;
     }
 
+    public function hasConnectedOtherIdp(): bool
```

Comment:
I love these simple methods, immediatelly clear to me. As Martin Fowler said, "any fool can write code that machines understand, but only good programmers write code that humans understand"